Gemfile

```diff
@@ -138,35 +139,35 @@ gem "rails-i18n",           "5.1.1"
 gem "markerb",             "1.1.0"
 
 # Map
-gem "leaflet-rails",       "1.2.0"
+gem "leaflet-rails",       "1.3.1"
 
 # Parsing
 
 gem "nokogiri",          "1.8.2"
 gem "open_graph_reader", "0.6.2" # also update User-Agent in features/support/webmock.rb
 gem "redcarpet",         "3.4.0"
 gem "ruby-oembed",       "0.12.0"
-gem "twitter-text",      "1.14.7"
+gem "twitter-text",      "2.1.0"
 
 # RTL support
 
-gem "string-direction", "1.2.0"
+gem "string-direction", "1.2.1"
 
 # Security Headers
 
-gem "secure_headers", "3.7.1"
+gem "secure_headers", "5.0.5"
 
 # Services
 
-gem "omniauth",           "1.6.1"
+gem "omniauth",           "1.8.1"
 gem "omniauth-facebook",  "4.0.0"
 gem "omniauth-tumblr",    "1.2"
 gem "omniauth-twitter",   "1.4.0"
-gem "twitter",            "6.1.0"
+gem "twitter",            "6.2.0"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `omniauth-wordpress` should appear before `twitter`.